src/Components/ItemExpanded.js

```diff
@@ -1,16 +1,101 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
 import styled from 'styled-components';
+import Button from "./Button";
 
+const ItemExpanded = ({ findItem }) => {
 
-const ItemExpanded = () => {
-    <section className="item-expanded">
-        <img className="item-image-large" src="" alt=""/>
-        <h2>Title</h2>
-        <h3>amount</h3>
-        <p>description</p>
-        <p>crafter</p>
-        <button className="contact-button"><a href="mailto:someone@example.com">Contact Crafter</a></button>
-    </section>
+    const { craftId } = useParams()
+
+    const [item, setItem] = useState({})
+
+    useEffect(() => {
+        setItem(findItem(craftId))
+    },[item])
+
+    return (
+        <ItemExpandedSection className="item-expanded">
+            {item ? <>
+                <ItemImgDiv>
+                    <ItemExpandedImg className="item-image-large" src={require("../assets/myrlene-numa-SnITZTTeJVE-unsplash.jpg")} alt=""/>
+                </ItemImgDiv>
+                <ItemExpandedDiv>
+                    <CategoryP>{item.category}</CategoryP>
+                    <TitleAmountDiv>
+                        <H2>{item.name}</H2>
+                        <AmountP>Amount: {item.amount}</AmountP>
+                    </TitleAmountDiv>
+                    <DescP>{item.description}</DescP>
+                    {/* <button className="contact-button"><a href="mailto:someone@example.com">Contact Crafter</a></button> */}
+                    <Button name="Contact Crafter" link="" />
+                </ItemExpandedDiv>
+            </>
+            : <h1>Loading...</h1>}
```

Comment:
Nice error handling here. We'll expand on this later in case the user inputs a craft id that doesnt exist